vivisect/contrib/__init__.py

```diff
@@ -1,23 +0,0 @@
-import os
-import sys
```

Comment:
Do we want to leave the contribution folder here with this kisser and just clear it out??

Reply:
I've always seen (both personally and in other repos) contrib as a place for third party modules that we depend on. But I'd be for clearing it out and just reserving it for whatever a user wants. We do also have the concept of VIV_EXT_PATH for UI plugins, and I'd rather not have multiple places a user can load in their custom code, since that tends to get confusing on what goes where.

But for now, if we just leave it empty, I'd be down for that. So I'll add that.

Reply:
if a user installs vivisect using an egg/whl/etc. then the viv resources will end up in `*-packages` somewhere in the python directories. i don't typically consider this area a place for users to touch. if there's an extension directory, i don't think it should be within the vivisect source tree. env var to specify `VIV_EXT_PATH` for extensions makes sense to me.

though, really, i'd recommend real virtualenvs for workspaces rather than messing with `sys.path`.

Reply:
One day. One day vivisect will be pip installable. I can dream.

Until then, that is a fair point. I've held off on adding contrib back in since I'd like to get all of us on the same page.